scrabble.js

```diff
@@ -1,8 +1,107 @@
 var Scrabble = function() {};
 
-// YOUR CODE HERE
-Scrabble.prototype.helloWorld = function() {
-  return 'hello world!';
+// setup score key
+Scrabble.prototype.score_key = {
+  1:  ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+  2:  ["D", "G"],
+  3:  ["B", "C", "M", "P"],
+  4:  ["F", "H", "V", "W", "Y"],
+  5:  ["K"],
+  8:  ["J", "X"],
+  10: ["Q", "Z"]
```

Comment:
I wish I had thought of setting up a dictionary when I did this project.  Would have made things so much simpler 😄  Excellent job!